.travis.yml

```diff
@@ -0,0 +1,95 @@
+language: cpp
+cache: ccache
+os:
+  - linux
+  - osx
+dist: xenial
+osx_image: xcode10.1
+compiler:
+  - clang
+  - gcc
+addons:
+  apt:
+    packages:
+      - cmake
+      - doxygen
+      - libpng-dev
+      - libxml2-utils
+      - llvm
+      - clang
+      - docutils-common
+      - libomp-dev
+      - lcov
+  homebrew:
+    packages:
+      - doxygen
+      - glib
+      - libpng
+      - libxml2
+      - ccache
+      - libomp
+notifications:
+  webhooks:
+    - https://coveralls.io/webhook
+env:
+  global:
+    - COVERALLS_PARALLEL=true
+  matrix:
+    - build_type=release
+    - build_type=release CMAKE_EFLAGS="-DBUILD_STATIC=ON -DBUILD_LENSTOOL=ON"
+    - build_type=debug
+    - build_type=debug CMAKE_EFLAGS="-DBUILD_STATIC=ON -DBUILD_LENSTOOL=ON"
+    - build_type=Coverage
+stages:
+  - name: test
+  - name: docs
+before_install:
+  - "sudo chown -R travis: $HOME/.ccache"
+  - mkdir -p $TRAVIS_BUILD_DIR/build
+install:
+  - pip install --user cpp-coveralls docutils
+  - export PATH=/Users/travis/Library/Python/2.7/bin:${PATH}
```

Comment:
Use Python 3. Python 2 is out of support soon.

Reply:
That's where Mac installs coveralls, I have not checked if they have python3

Reply:
Alright, Mac and Python is tricky on Travis CI. This will have to change at some point, but it's too much effort now to install a separate Python 3 via Homebrew or otherwise.